presidio_evaluator/models/flair_model.py

```diff
@@ -33,7 +33,6 @@ def __init__(
         verbose: bool = False,
```

Comment:
changes here: formatting and added `batch_predict` as it was added to the base class